public/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-Agent: * 
+Disallow: *
+Allow: /
```

Comment:
Sure about the order?

Reply:
https://developers.google.com/search/docs/advanced/robots/robots_txt#order-of-precedence-for-rules

> In case of conflicting rules, including those with wildcards, Google uses the least restrictive rule.

The issue is that I don't know how to test it :/.

Reply:
Good for me!